src/app/content/utils/urlUtils.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('getUrlParamForPageId', () => {
   });
 
   it('returns uuid param if tree is missing a slug', () => {
-    delete book.tree.contents[0].slug;
+    delete (book as any).tree.contents[0].slug;
```

Comment:
why is this necessary?

Reply:
`slug` isn't declared as optional so you can't `delete` it